tests/providers/amazon/aws/operators/test_sagemaker_training.py

```diff
@@ -127,3 +129,26 @@ def test_operator_defer(self, mock_training):
         with pytest.raises(TaskDeferred) as exc:
             self.sagemaker.execute(context=None)
         assert isinstance(exc.value.trigger, SageMakerTrigger), "Trigger is not a SagemakerTrigger"
+
+    @mock.patch.object(SageMakerHook, "describe_training_job")
+    @mock.patch.object(SageMakerHook, "create_training_job")
+    def test_execute_openlineage_data(self, mock_training, mock_desc):
+        mock_training.return_value = {
+            "TrainingJobArn": "test_arn",
+            "ResponseMetadata": {"HTTPStatusCode": 200},
+        }
+        mock_desc.return_value = {
+            "InputDataConfig": [
+                {
+                    "DataSource": {"S3DataSource": {"S3Uri": "s3://input-bucket/input-path"}},
+                }
+            ],
+            "ModelArtifacts": {"S3ModelArtifacts": "s3://model-bucket/model-path"},
+        }
+        self.sagemaker.check_if_job_exists = False
+        self.sagemaker._check_if_job_exists = mock.MagicMock()
```

Comment:
you could set this up in an annotation like the other 2

Reply:
Done.